dev-packages/node-integration-tests/suites/tracing/meta-tags-twp-errors/test.ts

```diff
@@ -17,17 +17,44 @@ describe('errors in TwP mode have same trace in trace context and getTraceData()
           const traceData = contexts?.traceData || {};
 
           expect(traceData['sentry-trace']).toEqual(`${trace_id}-${span_id}`);
+
           expect(traceData.baggage).toContain(`sentry-trace_id=${trace_id}`);
+          expect(traceData.baggage).not.toContain('sentry-sampled=');
 
           expect(traceData.metaTags).toContain(`<meta name="sentry-trace" content="${trace_id}-${span_id}"/>`);
-          expect(traceData.metaTags).toContain(`sentr y-trace_id=${trace_id}`);
```

Comment:
how did that pass before? 😅 oops

Reply:
yeah, it "passed" because the test function was async but it should have been sync and passed `done` to the runner. So I guess the assertion failed eventually but at that point the test already passed and we never saw any failing logs 😬